astropy/io/fits/hdu/groups.py

```diff
@@ -503,7 +503,7 @@ def _verify(self, option='warn'):
 
         self.req_cards('GCOUNT', pos, _is_int, 1, option, errs)
         self.req_cards('PCOUNT', pos, _is_int, 0, option, errs)
-        self.req_cards('GROUPS', pos, lambda v: (v == True), True, option,
+        self.req_cards('GROUPS', pos, lambda v: (v), True, option,
```

Comment:
This slightly worries me: what can `v` be? If it is guaranteed never to be an array, then this is OK (though parentheses can be removed). cc: @embray.